crates/nu-cli/tests/commands/merge.rs

```diff
@@ -0,0 +1,45 @@
+use nu_test_support::fs::Stub::FileWithContentToBeTrimmed;
+use nu_test_support::playground::Playground;
+use nu_test_support::{nu, pipeline};
+
+#[test]
+fn row() {
+    Playground::setup("merge_test_1", |dirs, sandbox| {
+        sandbox.with_files(vec![
+            FileWithContentToBeTrimmed(
+                "caballeros.csv",
+                r#"
+                name,country,luck
+                Andrés,Ecuador,0
+                Jonathan,USA,0
+                Jason,Canada,0
+                Yehuda,USA,0
+            "#,
+            ),
+            FileWithContentToBeTrimmed(
+                "new_caballeros.csv",
+                r#"
+                name,country,luck
+                Andrés Robalino,Guayaquil Ecuador,1
+                Jonathan Turner,New Zealand,1
+                Jason Gedge,Canada,0
+                Yehuda Katz,USA,0
+            "#,
+            ),
+        ]);
+
+        let actual = nu!(
+            cwd: dirs.test(), pipeline(
+            r#"
+                open caballeros.csv
+                | merge { open new_caballeros.csv }
```

Comment:
I'm trying to learn how `merge` works. It's not clear from this test what is happening (eg you could just `open new_caballeros.csv` rather than using `merge` and get the same result)

Reply:
Let's say you have a file `andres.csv` with:

```
name
andres
jonathan
```

let's say you have `jonathan.csv' with

```
policy
nush1
nush2
```

`open andres.csv | merge { open jonathan.csv }`

```
λ open andres.csv | merge { open jonathan.csv }
───┬──────────┬────────
 # │ name     │ policy
───┼──────────┼────────
 0 │ andres   │ sh1
 1 │ jonathan │ sh
───┴──────────┴────────
```